src/Components/Index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import Tasklist from "./Tasklist";
+import Add from "./Svg/add.svg";
+import AddTask from "./AddTask";
+
+const Index = (props) => {
+  const [showForm, setShowForm] = useState(false);
+  const tasks = props.items;
+  const completed = tasks.map((task) => {
+    return task.isCompleted && <Tasklist value={task} />;
```

Comment:
```suggestion
    return task.isCompleted && <TaskItem value={task} />;
```
I think `TaskItem` is more relevant then `TaskList` here.